logging/bin/deploy_esexporter.sh

```diff
@@ -66,6 +66,18 @@ else
   wnpValuesFile="$TMP_DIR/empty.yaml"
 fi
 
+# Deploying on OpenShift?
+OPENSHIFT_CLUSTER=${OPENSHIFT_CLUSTER:-false}
```

Comment:
I believe `OPENSHIFT_CLUSTER` should always be set now, so maybe this isn't needed

Reply:
removed the comment and line.